.zip-contents

```diff
@@ -14,3 +14,4 @@ config.xml
 LICENSE
 logo.png
 ps_accounts.php
+composer.json
```

Comment:
we should not include composer.json as it's more related to the build process, build zip will just need autoload files to work properly.

Reply:
Not sure of this, but I do think as for Node.js when a manifest file is always welcome in the resulting bundle to help people find useful info about the package author and such things. More like an info file, because no README file is either provided. What do you think?

Reply:
I get your intention, but in our case, the composer.json by itself is even lacking the scoping process to be understandable.

Reply:
Fine, I remove this.